### DISCUSSION TITLE
Tabs defaultActiveKey init failed. 挂载时初始化选中tab没有生效. 求答疑.

### DISCUSSION THREAD

**Discussion by transon-top** [ant-design/ant-design#36078] [Q&A] [answered]
Repetition of case 复现案例: 

https://stackblitz.com/edit/react-8cscna?file=demo.js

thanks
感谢

**Comment by huntdream** [comment, marked as answer]
把`defaultActiveKey` 改成`activeKey`

**Reply by transon-top** [reply]
感谢解答, activeKey 配合 onTabClick 中的传参key使用useState就可以受控了.
